<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>convert_crossword_type</class>
 <widget class="QWidget" name="convert_crossword_type">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>329</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>428</width>
    <height>329</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Convert To Crossword Type</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QToolBox" name="toolBox">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="pageConvertFrom">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>406</width>
           <height>185</height>
          </rect>
         </property>
         <attribute name="label">
          <string notr="true">Convert From: X</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="CrosswordTypeWidget" name="typeInfoFromWidget" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageConvertTo">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>406</width>
           <height>185</height>
          </rect>
         </property>
         <attribute name="label">
          <string notr="true">Convert To: X</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="KComboBox" name="crosswordType">
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="duplicatesEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="CrosswordTypeWidget" name="typeInfoToWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpConversionInfo">
     <property name="title">
      <string>Conversion Info</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="conversionInfo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string notr="true">Conversion info description</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CrosswordTypeWidget</class>
   <extends>QWidget</extends>
   <header>dialogs/crosswordtypewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>crosswordType</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
